File: server-mh/src/Match/Handler/Models/PlayerPathConfig.ts
export class PlayerPathConfig {
    public initialCells: number[];
    public startHomeEntryCell: number;
    public homeCells: number[];
    

    constructor(
        initialCells: number[] = [],
        startHomeEntryCell: number = 0,
        homeCells: number[] = [],
        
    ) {
        this.initialCells = initialCells;
        this.startHomeEntryCell = startHomeEntryCell;
        this.homeCells = homeCells;
      
    }
}

File: Client-unity/Ludo/Assets/Scripts/OfflineMatchHandler/PlayerPathConfig.cs
namespace Ludo.Offline
{
    public class PlayerPathConfig
    {
        public int[] InitialCells;
        public int StartHomeEntryCell;
        public int[] HomeCells;

        public PlayerPathConfig(
            int[] initialCells = null,
            int startHomeEntryCell = 0,
            int[] homeCells = null)
        {
            InitialCells = initialCells ?? new int[0];
            StartHomeEntryCell = startHomeEntryCell;
            HomeCells = homeCells ?? new int[0];
        }
    }
}